src/amo/components/UserProfile/styles.scss

```diff
@@ -20,10 +20,15 @@ $avatar-size: 64px;
 }
 
 .UserProfile-avatar {
-  display: block;
   height: $avatar-size;
   margin: 0 0 12px;
+  overflow: hidden;
   width: $avatar-size;
+
+  img {
+    width: 100%;
```

Comment:
Nitpick here, but our attributes are usually sorted alphabetically (I thought the linter would catch it 🤷‍♂️). It makes them easier to scan.